src/locale/deviceLocales.ts

```diff
@@ -41,17 +41,13 @@ export function getLocales() {
     }
 
     if (typeof locale.languageTag === 'string') {
-      if (locale.languageTag.startsWith('zh-Hans')) {
-        // Simplified Chinese to zh-CN
-        locale.languageTag = 'zh-CN'
+      if (locale.languageTag === 'zh-CN') {
```

Comment:
To ensure browser compatibility, the current versions of Chrome and Firefox still use RFC 1766 and return `zh-CN`.